test_pull_requests

```diff
@@ -53,7 +55,9 @@ opts = GetoptLong.new(
 ["--help",               "-h",      GetoptLong::NO_ARGUMENT],
 ["--merge_pull_request",            GetoptLong::REQUIRED_ARGUMENT],
 ["--local_merge_pull_request",      GetoptLong::REQUIRED_ARGUMENT],
+["--base_commit",                   GetoptLong::REQUIRED_ARGUMENT],
 ["--test_merge_pull_request",       GetoptLong::REQUIRED_ARGUMENT],
+["--can_skip_merge_tests",          GetoptLong::REQUIRED_ARGUMENT],
```

Comment:
Should the presence of the config in the `.test_pull_requests.json` be enough to indicate that we want this behavior?